docs/adguard-for-android/solving-problems/https-certificate-for-rooted.md

```diff
@@ -0,0 +1,66 @@
+
+However, on rooted devices, you can install the certificate to the **System store** and allow HTTPS filtering of all other apps' traffic too. 
+
+Here's how to do that.
+
+## How to install AdGuard Certificate to System store (on a rooted device)
+
+1. Enable HTTPS filtering in AdGuard for Android and save AdGuard's certificate to the User store (use [this instruction](../../overview#https-filtering) if needed)
+2. Go to **AdGuard app** > **Menu** (≡) > **Settings** > **Network** > **HTTPS filtering** > **Secure certificate** >
+tap “**Copy to the system store**”
+
+That is enough for older versions of Magisk.
+
+However, if you have a newer version, you will get this message:
+
+> Unable to copy the certificate to the system store. Try using “AdGuard Certificate” module.
+
+In that case, proceed to steps below:
+
+3. Go to **Magisk** -> **Settings**
+
+![Open Magisk settings *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-1.png)
+
+4. Enable **Zygisk**
+
+![Enable Zygisk *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-2.png)
+
+![Go back to Magisk main screen *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-3.png)
+
+5. Download the `.zip` file (of “AdGuard Certificate” module) from the [latest release on GitHub](https://github.com/AdguardTeam/adguardcert/releases/latest/)
+
+6. Go to **Magisk** -> **Modules** -> **Install from storage** and select the downloaded `.zip` file
+
+![Open Magisk modules *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-4.png)
+
+![Install from storage *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-5.png)
+
+![Select AdGuard certificate module *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-6.png)
+
+7. Reboot
+
+![Reboot the device](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/https-certificate-for-rooted/magisk-module-7.png)
+
+If a new version of "AdGuard certificate" module comes out, repeat steps 3-7 to update the module.
+
+The module does its work during the system boot. If your AdGuard certificate changes,
+you'll have to reboot the device for the new certificate to be copied to the system store.
+
+### Bromite browser
+
+Please note that in order for **Bromite** browser to work properly, in addition to the steps mentioned above, you need to set flag "Allow user certificates" in `chrome://flags` to "Enabled" state.
+
+### Chrome and Chromium-based browsers
+
+Long story short, you will have no problems with HTTPS filtering in Chrome and Chromium-based browsers on rooted devices, if you use "AdGuard Certificate" module. 
```

Comment:
Should we explain flow with 2 certs here or is it too soon? @ameshkov

Reply:
Not yet as we don't yet support it in the stable version